### PR TITLE
feat: Add usage in readme

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

Add usage.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @ddysher @supereagle @codeflitting 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,124 +2,86 @@
...
+│       ├── variables.data-00000-of-00001
+│       └── variables.index
+└── ormbfile.yaml
+
+2 directories, 4 files
+```
+
+## Community
+
+ormb project is a part of Clever, which is a Cloud Native Machine Learning platform. We are going to open source the platform soon.
```
Comment (ddysher) [inline README.md:85]:
```suggestion
ormb project is part of Clever, a Cloud Native Machine Learning platform. We are going to open source a community edition soon.
```

File: README.md [hunk 2]
```diff
@@ -2,124 +2,86 @@
...
-├── test
-│   ├── README.md
-│   └── test_make.sh
-├── third_party
-│   └── README.md
-└── vendor
-    └── README.md
+# ormb
+
+Manage Machine Learning/Deep Learning Models like Docker Images.
```
Comment (ddysher) [inline README.md:18]:
```suggestion
Managing Machine Learning/Deep Learning Models like Docker Images.
```

**Comment by gaocegege** [comment]
Comments are addressed and commits are rebased, PTAL

**Pull request merged**